src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -199,7 +199,13 @@ public enum MPSCnnNeuronType {
 		Elu,
 		PReLU,
 		ReLun,
-		Count,
+		[TV (11,3), Mac (10,13,4, onlyOn64: true), iOS (11,3)]
+		Power,
+		[TV (11,3), Mac (10,13,4, onlyOn64: true), iOS (11,3)]
+		Exponential,
+		[TV (11,3), Mac (10,13,4, onlyOn64: true), iOS (11,3)]
+		Logarithm,
+		Count, // must always be last
```

Comment:
That's problematic. Maybe we can replace it with an extension method that will always provide the correct value (length - 1).